tests/unit/hybrid_engine/test_he_lora.py

```diff
@@ -125,7 +126,8 @@ def get_model(self, model_name):
         model_config.dropout = 0.0
         model = AutoModelForCausalLM.from_pretrained(model_name, config=model_config)
         model = model.half()
-        model = model.to(f'cuda:{local_rank}')
+        device = get_accelerator().device_name()
```

Comment:
I'm not sure the hybrid engine currently supports anything other than Cuda and only certain devices at the moment, @lekurile to confirm?

Reply:
@loadams We added support for Hybrid Engine via our fork, using the existing fallback python implementation to the cuda kernels within the injected layers. 
We successfuly able to train deepspeed-chat step3 on llamav2-7b using HE.
We are in the process of PRing it - as it requires some cosmetics and generalization.